components/atom/input/src/Input/Component/index.js

```diff
@@ -107,12 +111,15 @@ const Input = forwardRef(
         required={required}
         pattern={pattern}
         inputMode={inputMode}
+        children={as === 'input' ? undefined : children}
```

Comment:
let children always become added but normally not used to simplify the logic and add it not as a prop but as a nested element

Reply:
I needed to do it this way because there was an error related to the input (it can not have children)